.werft/installer-tests.ts

```diff
@@ -268,10 +268,11 @@ if (config === undefined) {
 
 installerTests(TEST_CONFIGURATIONS[testConfig]).catch((err) => {
     if(deleteOnFail == "true") {
-        cleanup();
+        cleanup().finally(() => {
+            console.error(err);
+            process.exit(1);
+        });
     }
-    console.error(err);
```

Comment:
I feel like this should still be there. Even if we are not explicitly asking to fail, I believe if an error is quote it should respond with a process.exit (failed job)

Reply:
See this example https://werft.gitpod-dev.com/job/gitpod-custom-alt-stream-exec-logs.5

Reply:
I agree - but because cleanup is now async the old code won't work anymore though - so it could be something like the following

```ts
if(deleteOnFail == "true") {
    cleanup().finally(() => {
        console.error(err);
        process.exit(1);
    });
} else {
    console.error(err);
    process.exit(1);
}
```

Reply:
Resolved in b84a84dfc3389e4a2d5c03e4c960a4e4ee22f65a